src/css/module.scss

```diff
@@ -245,4 +245,41 @@ body #forien-quest-log .actions i.move.fas.fa-stop-circle:hover, body .window-ap
 body #forien-quest-log .quest-log .table ul .title p {
     color: $white;
 }
-//End Quest Log Styles
\ No newline at end of file
+//End Quest Log Styles
+
+//Start Simple Calendar
+/**
+ * CHAT MESSAGE STYLING
+ */
+// Public Roll
+.chat-message,
```

Comment:
While this is intended for a single module, the classes seem to target global Foundry classes only.

It would be best to limit module specific css classes to only affect that modules window (if possible). Doesn't have simple calendar a per app class, similar to .sr5 for the system.

Reply:
Thses are the classes used by simple calendar for the chat card

Reply:
.chat-message is a bit of a difficult area since it's the default Foundry class for chat messages. So all modules writing default chat messages will use that class.

I'll test a few chat messages sources to make sure it's not conflicting with normal stuff (system test and item chat card messages).

After that I'll merge :)

Thank you again for your help!

Reply:
The chat cards and the test prompt windows are my next look I will do, on a rework to get them consistent.